miniai/sgd.py

```diff
@@ -30,7 +30,9 @@
 # %% ../nbs/12_accel_sgd.ipynb 44
 class BaseSchedCB(Callback):
     def __init__(self, sched): self.sched = sched
-    def before_fit(self, learn): self.schedo = self.sched(learn.opt)
+    def before_fit(self, learn): 
+        self.schedo = self.sched(learn.opt)
+        learn.schedo = self.schedo
```

Comment:
What's this for?

Reply:
To have access to the scheduler inside the Learner object. This way, I can monitor the learning rate or other scheduler params.

Reply:
Can you do that using the optimiser hyperparams instead?

Reply:
oh, you are right =)

Reply:
`opt.param_groups[0]["lr"]` returns the current learning rate from the `optimizer` object